native_client/definitions.mk

```diff
@@ -78,7 +78,7 @@ PYVER := $(shell python -c "import platform; maj, min, _ = platform.python_versi
 PYTHON_PACKAGES      :=
 PYTHON_PATH          := PYTHONPATH=$(RASPBIAN)/usr/lib/python$(PYVER)/:$(RASPBIAN)/usr/lib/python3/dist-packages/
 NUMPY_INCLUDE        := NUMPY_INCLUDE=$(RASPBIAN)/usr/include/python3.7m/
-PYTHON_SYSCONFIGDATA := _PYTHON_SYSCONFIGDATA_NAME=_sysconfigdata_m_linux_arm-linux-gnueabihf
+PYTHON_SYSCONFIGDATA := _PYTHON_SYSCONFIGDATA_NAME=_sysconfigdata_m_linux_arm-linux-gnu
```

Comment:
@lissyx not sure if this is the correct value `rpi3 ` but I have debug statement to find out

Reply:
I dont think you should remove `gnueabihf` in favor of `gnu`, but that might depend on the python versions you are addressing ...

Reply:
I mean there's a reason we sticked to only python versions supported by the current stable version of Raspbian :)

Reply:
@lissyx hmm the build for python 3.7 aarch seems to be working fine. However for python 3.8 is failing:

https://github.com/coqui-ai/STT/runs/7586202693?check_suite_focus=true

not quite sure why since the module seems to exist. By any chance do you know what could possibly be wrong?

Reply:
Check the result of your find ? The error is pretty clear: `ModuleNotFoundError: No module named '_sysconfigdata_m_linux_aarch64-linux-gnu'`

Reply:
yeah find is pointing to another one:

```
Run python3 --version
Python 3.8.13
pip 22.2.1 from /opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/pip (python 3.8)
/usr/lib/python3.8/_sysconfigdata__x86_64-linux-gnu.py
```

Reply:
https://github.com/coqui-ai/STT/runs/7586202693?check_suite_focus=true#step:14:26 I think your command is not ran against the chroot but against the host ...

`find ${SYSTEM_RASPBIAN}/usr/lib/python3.*/ -type f -name "*.py" | grep sysconfigdata_` maybe?

Reply:
perfect trying again :)